action.yml

```diff
@@ -1,10 +1,9 @@
 name: Slate documentation builder
 author: Decathlon <developers@decathlon.com>
-description: This Github action <strong>builds repositories Markdown files</strong> using slate framework.
-The slate build result will be stored into the build subdirectory of <md files folder>.
+description: This Github action <strong>builds repositories Markdown files</strong> using slate framework. The slate build result will be stored into the build subdirectory of <md files folder>.
 runs:
   using: 'docker'
-  image: 'docker://decathlon/slate-builder-action:2.0.0'
+  image: 'Dockerfile'
```

Comment:
Not sure about this. In this way, when we trigger the CI on this repository we are able to use the action directly on the branch or PR because it is using the local Dockerfile.
@lusoalex @rdelgatte What is the final purpose of the `action.yml` file? The submit to the marketplace? Do we have to show always the latest available docker image? Do we need to update this file manually anytime a new version is pushed?